### DISCUSSION TITLE
Sleeping notebook and Calibration Mode

### DISCUSSION THREAD

**Discussion by RoHe56** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#944] [Q&A] [answered]
1. Is it possible to perform Calibration Mode even if the laptop is in sleeping mode and the "Stop Charging When Sleeping" function is activated ?
2. Is it reasonable to discharge the battery to 0% in Calibration Mode as you state? As far as I know, Lion batteries should not be discharged this way.

Thank you Roman

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @RoHe56,

> 1. Is it possible to perform Calibration Mode even if the laptop is in sleeping mode and the "Stop Charging When Sleeping" function is activated ?

No, while the Calibration Mode is running, AlDente will disable the sleep mode in order to run the Calibration Mode without interruptions. You can, however, have the "Stop Charging When Sleeping" function enabled, since AlDente will override this function while in Calibration Mode.

> 2. Is it reasonable to discharge the battery to 0% in Calibration Mode as you state? As far as I know, Lion batteries should not be discharged this way.

In general, you should avoid using your MacBook with a battery charge level below 20%. However, using the Calibration Mode every other week will not significantly damage the battery and has the benefit of keeping the battery calibrated. Furthermore, the Calibration Mode will discharge your MacBook only to 15%.

Cheers,
Matthias

**Reply by RoHe56** [reply]
I understand that it is possible to perform Calibration Mode  automatically while the laptop is sleeping, e.g. at night. OK ?

Thanks Roman

**Reply by MatthiasKerbl** [reply]
Yes, you can let the Calibration Mode run over night and even close the lid of your MacBook. You only need to let the MacBook be plugged in.

Cheers,
Matthias
